### ISSUE TITLE
browser.getMetrics is not a function [DevTools service]

### ISSUE THREAD

**Issue by gtreskas** [webdriverio/webdriverio#5405]
Same example as already fixed with issue https://github.com/webdriverio/webdriverio/issues/4278, is now reoccurring again.

**Environment (please complete the following information):**
 - **WebdriverIO version:** 6.1.11
 - **Mode:** Local WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** async
 - **Node.js version:** 12.16.3
 - **NPM version:** 6.13.2
 - **Browser name and version:** Chrome 81
 - **Platform name and version:** Windows 10
 - **Additional wdio packages used (if applicable):**  DevTools, ChromeService, Spec Report

**Config of WebdriverIO**
See example https://github.com/webdriverio/webdriverio/issues/4278
**Describe the bug**
When running the test with the configuration and spec specified:
https://github.com/webdriverio/webdriverio/issues/4278
A message appears:  browser.getMetrics is not a function

**Logs:**
TypeError: browser.getMetrics is not a function
[chrome  windows #0-0]     at Context.<anonymous> (C:\webdriverio\vyper-wdio\tests\vyper\nonFunctional\performance\dummyPerf.spec.js:13:19)
[chrome  windows #0-0]     at async Context.executeAsync (C:\webdriverio\vyper-wdio\node_modules\@wdio\utils\build\shim.js:123:12)
[chrome  windows #0-0]     at async Context.testFrameworkFnWrapper (C:\webdriverio\vyper-wdio\node_modules\@wdio\utils\build\test-framework\testFnWrapper.js:57:14)

**To Reproduce**
Execute the script in https://github.com/webdriverio/webdriverio/issues/4278.

**Expected behavior**
The metrics of performance should be shown.

**Comment by christian-bromann** [comment]
It seems that WebdriverIO is not able to connect to the browser. We already logged [an issue](https://github.com/webdriverio/webdriverio/issues/5374) for this to be more expressive in the errors about this. I tried to reproduce this with my [demo project](https://github.com/christian-bromann/webdriverio-performance-testing) and had no issues after all. The only problem I ran into was that Chrome wasn't able to open my page due to SSL issues. I added `acceptInsecureCerts: true` as a workaround for it.
